.ko.yaml

```diff
@@ -0,0 +1,62 @@
+#
```

Comment:
Just wondering cause I'm not familiar, is `.ko` just used when running the `ko` commands from the Makefile locally? Or used elsewhere?

Reply:
that is used when `ko` run , then it reads the `.ko.yaml` config and use that to build the image